src/EventStore.ClientAPI/EventStore.ClientAPI.csproj

```diff
@@ -11,11 +11,17 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
     <DocumentationFile>$(OutputPath)\EventStore.ClientAPI.xml</DocumentationFile>
   </PropertyGroup>
-  <ItemGroup>
-    <Reference Include="System.Net.Http" />
-  </ItemGroup>
   <ItemGroup>
       <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
       <PackageReference Include="protobuf-net" Version="2.4.0" />
   </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net46' ">
+    <Reference Include="System.Net.Http" />
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
+      <PackageReference Include="System.Net.Http" Version="4.3.4" />
+      <PackageReference Include="System.Net.Requests" Version="4.3.0" />
+      <PackageReference Include="System.Net.Security" Version="4.3.2" />
+      <PackageReference Include="System.Data.SqlClient" Version="4.5.1" />
```

Comment:
if we change the documentation comment on the EventStoreConnection that refers to SqlConnection using <seealso /> then we can remove our dependency on the SqlClient,.

Reply:
@jageall fixed thanks!